### DISCUSSION TITLE
Close Command Palette with keyboard

### DISCUSSION THREAD

**Discussion by Industrial** [community/community#15696] [Other Feature Feedback, Questions, & Ideas]
Hi!

I think the Command Palette is a great feature. I would like to close the palette with the keyboard. Intuitively I tried to use escape to close it but it didn't work.

gr,

Tom

**Comment by mykhaliuk** [comment]
Thanks for a great feature!

Totally agree with @Industrial. It would be nice to have the possibility to close the Palette with an `Esc` key

**Comment by ecstrema** [comment]
Esc works for me in all browsers
